lib/nerves_hub_link_common/downloader.ex

```diff
@@ -18,7 +18,11 @@ defmodule NervesHubLinkCommon.Downloader do
   require Logger
   use GenServer
 
-  alias NervesHubLinkCommon.{Downloader, Downloader.RetryConfig, Downloader.TimeoutCalculation}
+  alias NervesHubLinkCommon.{
```

Comment:
I think the Elixir formatter is playing games with you since this line shouldn't have changed.

Reply:
i think this is related to my rebase, removing the retryconfig commit. fixing..